src/es/home.html

```diff
@@ -0,0 +1,121 @@
+
+      <dt>Igualdad en aplicaciones web</dt>
+      <dd><strong>Se deben eliminar todas las barreras artificiales colocadas por los guardianes.</strong> Las aplicaciones web, si se permiten, pueden ofrecer una funcionalidad equivalente con mayor privacidad y seguridad para casos de uso exigentes.</dd>
+    </dl>
+  </div>
+</div>
+
+<div class="get-involved">
+  <div class="who-entry">
+    <div class="group">
+      <h2>Apoyanos</h2>
+      <p>Los reguladores del Reino Unido, la UE, Japón, Australia y los EE. UU. han comenzado a investigar la prohibición de Apple de los motores de búsqueda de terceros y están considerando intervenciones para permitir que las aplicaciones web compitan con las nativas. Lo más importante es que se ha incluido un texto en este sentido en la histórica ley tecnológica de la UE, la Ley de Mercados Digitales.</p>
+
+      <p>Esto no era inevitable. Durante los últimos dos años, Open Web Advocacy ha informado ampliamente a los reguladores y legisladores sobre el potencial de la Web para contrarrestar el duopolio de las aplicaciones móviles.</p>
+
+      <p>Casi con toda seguridad, como resultado directo de nuestra defensa, Apple ha contratado a docenas de empleados adicionales
+para trabajar en WebKit y Safari. Este ancho de banda adicional y la presión regulatoria han contribuido directamente
+a la incorporación de funciones que se habían demorado durante mucho tiempo, como las notificaciones push y las credenciales de aplicaciones para las aplicaciones web.
+Las cifras del personal de WebKit son secretas, pero creemos que se trata de un aumento porcentual significativo.</p>
+
+      <p><strong>La competencia, o al menos la amenaza de ella, funciona.</strong></p>
+
+      <p>OWA necesita tu ayuda para seguir ejerciendo presión para restablecer la competencia y hacer realidad
+las aplicaciones web universales. Puedes marcar una verdadera diferencia para garantizar el futuro de la
+web apoyando esta labor. Dependemos de las donaciones para desafiar a los guardianes increíblemente ricos
+que tienen todos los incentivos para frenar el avance de la web.</p>
+
+      <p>Puedes<a href="/donate">donar</a> o únete a nuestra comunidad en
+        <a href="https://discord.com/invite/x53hkqrRKx">Discord</a> para involucrarte más profundamente. ¿No puedes prescindir del
+dinero ni el tiempo? Manténgate al tanto de lo que hacemos y ayude a difundir la información siguiéndonos en
+        <a href="https://twitter.com/OpenWebAdvocacy">Twitter</a> y/o <a href="https://mastodon.social/@owa">Mastodon</a>
+        .</p>
+    </div>
+
+    <div class="prom-banner">
+      <p class="illustration"><img src="/images/donate.svg" alt="" /></p>
+      <p>Si te gusta lo que hacemos y te gustaría apoyar nuestro trabajo, por favor considera
+        <a href="https://www.paypal.com/donate/?hosted_button_id=3FD5DUWT4DNBG">haciendo una donación</a> o consiguiendo que tu empresa nos apoye.</p>
```

Comment:
```suggestion
        <a href="https://www.paypal.com/donate/?hosted_button_id=3FD5DUWT4DNBG">hacer una donación</a> o conseguir que tu empresa nos apoye.</p>
```
I think that in English that "present continuous", right? Well, it doesn't work the same way in Spanish